setup-git.sh

```diff
@@ -15,26 +15,33 @@ mkdir -p ${GITOPS_DIR}
 gitops_workload_url="$(terraform -chdir=${ROOTDIR}/terraform/codecommit output -raw gitops_workload_url)"
 gitops_platform_url="$(terraform -chdir=${ROOTDIR}/terraform/codecommit output -raw gitops_platform_url)"
 gitops_addons_url="$(terraform -chdir=${ROOTDIR}/terraform/codecommit output -raw gitops_addons_url)"
+gitops_fleet_url="$(terraform -chdir=${ROOTDIR}/terraform/codecommit output -raw gitops_fleet_url)"
```

Comment:
You remove workloads elsewhere but not here. Not sur how fleet replace workloads ?